universum/modules/launcher.py

```diff
@@ -350,30 +357,27 @@ def __init__(self, *args, **kwargs):
         self.api_support = self.api_support_factory()
         self.reporter = self.reporter_factory()
         self.server = self.server_factory()
-        self.code_report_collector = self.code_report_collector()
+        self.code_report_collector = self.code_report_collector_factory()
         self.include_patterns, self.exclude_patterns = get_match_patterns(self.settings.step_filter)
 
     @make_block("Processing project configs")
-    def process_project_configs(self):
+    def process_project_configs(self) -> configuration_support.Variations:
         config_path = utils.parse_path(self.settings.config_path, self.settings.project_root)
         configuration_support.set_project_root(self.settings.project_root)
-        config_globals = {}
+        config_globals: Dict[str, configuration_support.Variations] = {}
```

Comment:
I am not sure regarding technical consequences of this declaration, but config_globals is just a dictionary of all global variables of the configs module. Potentially it can contain variable of any type.

Reply:
sure, I can remove this part, and add explicit type cast, where Variations are retrieved from dict